sharding/config_test.go

```diff
@@ -36,7 +36,7 @@ func TestMinProposerBalance(t *testing.T) {
 }
 
 func TestNotarySubsidy(t *testing.T) {
-	want, err := new(big.Int).SetString("1000000000000000", 10) // 0.001 ETH
+	want, err := new(big.Int).SetString("1000000000000000", 10) // 0.001 ETH.
```

Comment:
I don't think the punctuation is always necessary, only when it makes sense such as in a sentence. 

With that said, this is still fine too